.freeCodeCamp/tooling/lesson.js

```diff
@@ -54,13 +54,15 @@ async function runLesson(ws, projectDashedName) {
   updateDescription(ws, description);
 
   const seed = getLessonSeed(lesson);
-  const isForce = isForceFlag(seed);
-  // force flag overrides seed flag
-  if (
-    (project.seedEveryLesson && !isForce) ||
-    (!project.seedEveryLesson && isForce)
-  ) {
-    await seedLesson(ws, project, lessonNumber);
+  if (seed) {
+    const isForce = isForceFlag(seed);
```

Comment:
What is this isForce for?

Reply:
Sometimes, I feel you do not listen to any of my demonstrations 😢 

You can use the `#### --force--` flag to override the behaviour of:
a) Lesson automatically seeds on next lesson - force stops this
b) Lesson never seeds on next lesson - force causes the seed to be run when a Camper lands on the next lesson.